### PR TITLE
Add template for incident report creation under docs directory

### PR DESCRIPTION
**Pull request by balajialg**
Fixes one of the next steps from #3983

### CODE REVIEW

File: docs/admins/incidents/2022-12-16-incident-report-template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Eg: 2018-02-28 - A node hangs, causing a subset of users to report issues
+
+##Summary
+
+Explain briefly the issue, its impact and the short term solution to fix the issue.
+
+## Timeline
+
+Sequence the series of events that led to this issue. 
+
```
Comment (ryanlovett) [inline docs/admins/incidents/2022-12-16-incident-report-template.md:11]: Perhaps add examples, e.g.

```
### 2025-01-05 11:00 AM
Something happened.

### 11:45 AM
Another thing happened on the same day.
```

**Review by ryanlovett — COMMENTED** [review]
This looks great! Just one minor suggestion.

This can be followed up by a separate PR which documents when and how to instantiate a new incident report.

I thought it'd be cool if this were implemented as a GitHub [pull request template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository), but it looks like this would be [the default template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates#pull-request-templates) for all PRs which is not what we want. A fancier but not quite necessary variant would be an issue template which solicits all of the same info along with a GitHub action that turns it into a PR.

**Comment by balajialg** [comment]
@ryanlovett I love the idea of creating a GitHub issue template that gets converted into a PR when we fill it. Can you point me to resources that I can work with to do this conversion?

I found this script that does the github issue template to PR conversion - https://github.com/jehiah/git-open-pull. However, this has to be run manually to do the conversion. I would love to find a tool that does issue templates to pull request conversion when information gets filled.

**Comment by ryanlovett** [comment]
@balajialg It would be something like https://github.com/marketplace/actions/create-pr or https://github.com/marketplace/actions/create-pr-action. Basically GitHub can be automated with Actions -- bits of configuration in a repo that define actions to take based on certain conditions. The first linked action will create a PR based on specified conditions. I would imagine it could create a PR based on if an issue was tagged with `#incident` or some such.

Here's more info:
https://github.com/features/actions
https://aiven.io/blog/how-to-automagically-open-pull-requests-with-github-actions

**Comment by balajialg** [comment]
Thanks @ryanlovett! I will look into this next week. I will also block sometime with you next week if I get stuck. Thanks

**Comment by balajialg** [comment]
Closing this PR as I don't have enough cycles to prioritize the optimizations to incident report template.

**Pull request closed without merging**